spec/components/covid_cta_spec.rb

```diff
@@ -0,0 +1,36 @@
+require "rails_helper"
+
+describe "CovidCta", type: :view do
+  def component_name
+    "contextual_sidebar/covid_cta"
+  end
+
+  before(:each) do
+    I18n.locale = :en
+  end
+
+  let(:content_item) do
+    {
+      "base_path" => "/any-page",
+      "content_id" => SecureRandom.uuid,
+    }
+  end
+
+  let(:welsh_content_item) { content_item.merge("locale" => "cy") }
```

Comment:
I asked Jessie if there will be a Welsh translation. She is checking with Ben, but that may be something that needs adding before the merge!

Reply:
I checked with Jessie. No Welsh translation.